Classifier_Klasse.py

```diff
@@ -12,14 +12,25 @@
 from Word_features import count_stopwords
 from exceptions import FeatureException
 from exceptions import InputException
+from Evaluierung_Klasse import Evaluation
 import warnings
 
 class Classifier():
 
     def __init__(self, df):
         self.df = df
 
-    def train_model(self, features_for_train):
+    def train_model(self, features_for_train):  
+        
+        ''' 
+        Zeit-Komplexität für diese Schleife ist O(max(N, M)))
+        Diese Schleife prüft, ob die von User gegebenen Features zugänglich
+        und schon berechnet sind. Wenn nicht, wirft die Schleife einen Fehler.
+        Die Schleife erstellt eine neue Liste mit zugänglichen Feautres. 
+        Man kann anstatt dieser Liste einen Generator benutzen,
+        um die Memory zu sparren. 
+        '''
```

Comment:
Immer dazu angeben, was M und N. Tatsächlich ist die Überprüfung `c not in list(self.df.columns)[1:]` in Python auch eine Aktion mit linearer Laufzeit also O(n) wobei n die Länge der List ist, in der nach einem Element gesucht wird. 🐞